tests/features/skos.feature

```diff
@@ -0,0 +1,37 @@
+@api
+Feature: SKOS feature
+  In order to work with vocabularies
+  As a site administrator
+  I should be able to see SKOS entities
+
+  @skos
```

Comment:
We can put this tag at the beginning of the feature.

Reply:
No we cannot, prefeature hooks use static methods so we don't have access to the rest of the context.